src/components/SearchSubthoughts.js

```diff
@@ -58,6 +59,7 @@ const SearchSubthoughts = ({ search, searchLimit = DEFAULT_SEARCH_LIMIT, dispatc
   const children = search ? rankThoughtsSequential(
     sort(Object.values(thoughtIndex)
       .filter(thought =>
+        (archived && isArchived(thought)) &&
```

Comment:
I think you want `archived || !isArchived(thought)`. Normal thoughts are getting excluded when `archived` is checked.

Reply:
820a2627